src/components/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import styles from "@/styles/Header.module.css";
+
+const Header = (props) => {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '1.5em',
+    color: '#aeadad',
+    textAlign: 'center',
+  };
+  return (
+    <header style={headerStyle} className={styles.header}>
+      {props.children}
```

Comment:
- [ ] You did a good job working on your Header.jsx 💯 However, please make sure that the {props.children} has the prop validation  kindly use the code below to fix it. 🤓 🔧 

Header.propTypes = {
  children: PropTypes.node.isRequired,
};